src/test/java/test/LogoutTest.java

```diff
@@ -0,0 +1,42 @@
+package test;
+
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.openqa.selenium.WebDriver;
+import pages.LoginPopup;
+import pages.LogoutPopup;
+import pages.WebDriverSingleton;
+
+import java.util.concurrent.TimeUnit;
+
+public class LogoutTest {
+
+    private static final String TEST_LOGIN = "seleniumtests@tut.by";
+    private static final String TEST_PASSWORD = "123456789zxcvbn";
+
+    WebDriver driver;
+    LoginPopup loginPopup;
+    LogoutPopup logoutPopup;
+
+    @BeforeEach
+    public void openBrowser() {
+        driver = WebDriverSingleton.getInstance();
+        driver.get("https://www.tut.by/");
```

Comment:
Please move this navigation to some method in LoginPopup page object class (f.e. load()). Do not forget to add returning value for new method + move string to constant